transportable-udfs-spark_2.11/src/test/scala/com/linkedin/transport/spark/data/TestSparkRowData.scala

```diff
@@ -25,43 +25,43 @@ class TestSparkRowData {
 
   @Test
   def testCreateSparkStruct(): Unit = {
-    val stdStruct = SparkConverters.toTransportData(structData, structType).asInstanceOf[RowData]
-    assertSame(stdStruct.asInstanceOf[PlatformData].getUnderlyingData, structData)
+    val rowData = SparkConverters.toTransportData(structData, structType).asInstanceOf[RowData]
```

Comment:
There are other places with `row` as opposed to `rowData`. Let us use `rowData`.

Reply:
ok

Reply:
addressed

Reply:
Could you update `array` and `map` to `arrayData` and `mapData`?

Reply:
please more specific for your request. I don't think there is any array and map in this TestSparkRowData class. And, arrayData and mapData are applied properly in the project.

Reply:
They are not applied properly in the project. Please take a look at other classes in this patch.